shell/platform/windows/keyboard_unittests.cc

```diff
@@ -316,76 +316,6 @@ class MockKeyboardManagerDelegate : public KeyboardManager::WindowDelegate,
   std::list<Win32Message> redispatched_messages_;
 };
 
-// A FlutterWindowsView that overrides the RegisterKeyboardHandlers function
-// to register the keyboard hook handlers that can be spied upon.
-class TestFlutterWindowsView : public FlutterWindowsView {
- public:
-  typedef std::function<void(const std::u16string& text)> U16StringHandler;
-
-  TestFlutterWindowsView(
-      U16StringHandler on_text,
-      KeyboardKeyEmbedderHandler::GetKeyStateHandler get_keyboard_state,
-      KeyboardKeyEmbedderHandler::MapVirtualKeyToScanCode map_vk_to_scan)
-      // The WindowBindingHandler is used for window size and such, and doesn't
-      // affect keyboard.
-      : FlutterWindowsView(
-            std::make_unique<::testing::NiceMock<MockWindowBindingHandler>>()),
-        get_keyboard_state_(std::move(get_keyboard_state)),
-        map_vk_to_scan_(std::move(map_vk_to_scan)),
-        on_text_(std::move(on_text)) {}
-
-  void OnText(const std::u16string& text) override { on_text_(text); }
-
-  void HandleMessage(const char* channel,
-                     const char* method,
-                     const char* args) {
```

Comment:
This helper sends a platform message and isn't actually related to the view in any way. I moved it to the `KeyboardTester` and renamed it `InjectPlatformMessage`.